libs/libc/string/CMakeLists.txt

```diff
@@ -119,6 +118,10 @@ if(NOT CONFIG_LIBC_ARCH_STRNLEN)
   list(APPEND SRCS lib_strnlen.c)
 endif()
 
+if(CONFIG_ALLOW_MIT_COMPONENTS)
+  list(APPEND SRCS lib_memmem.c)
```

Comment:
lib_strstr.c should also add to this license

Reply:
strstr is used in many places, I guess most defconfig need add CONFIG_ALLOW_MIT_COMPONENTS.